CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 3.0)
 
+# allow parent project to override options
```

Comment:
Nitpick: Would your project be the parent project or the _dependent_ project? Other than potentially updating this, your PR comment explains the need well. :)